tools/gyp/pylib/gyp/MSVSSettings.py

```diff
@@ -314,7 +314,14 @@ def _MSBuildOnly(tool, name, setting_type):
     name: the name of the setting.
     setting_type: the type of this setting.
   """
+
+  def _Translate(value, msbuild_settings):
+    # Let msbuild-only properties get translated as-is from msvs_settings.
+    tool_settings = msbuild_settings.setdefault(tool.msbuild_name, {})
```

Comment:
This line actually modifies `msbuild_settings` dictionary. Is that intentional? If only the default value is required, the correct way to do this is `msbuild_settings.get(tool.msbuild_name, {})`